cmake/catkin_install_python.cmake

```diff
@@ -19,14 +19,14 @@ function(catkin_install_python signature)
   if(NOT ARG_DESTINATION)
     message(FATAL_ERROR "catkin_install_python() called without required DESTINATION argument.")
   endif()
-  foreach(file ${ARG_UNPARSED_ARGUMENTS})
-    if(NOT IS_ABSOLUTE ${file})
-      set(file "${CMAKE_CURRENT_SOURCE_DIR}/${file}")
+  foreach(source_file ${ARG_UNPARSED_ARGUMENTS})
```

Comment:
Can you revert the renaming of this variable? I would expect this to make the PR much smaller and easier to review. (Or separate the change in a separate commit.)

Reply:
Split into two commits. The renaming happens in f9e2e6b